sources/simulators.swift

```diff
@@ -29,14 +53,10 @@ func getBootedSimulators() throws -> [String] {
     }
 
     let devices = json["devices"] as? [String: [[String: String]]] ?? [:]
-    let bootedIDs = devices
-        .flatMap { $1 }
-        .filter { $0["state"] == "Booted" }
-        .flatMap { $0["udid"] }
-
-    if bootedIDs.isEmpty {
+    let bootedSimulators = devices.flatMap { $1 }.flatMap(Simulator.init)
```

Comment:
could do similar `guard let ` or ` if let` here

Reply:
… I think the `state == .booted` check should be/stay here.

Reply:
moved it back here